src/util/fs.ts

```diff
@@ -47,3 +47,22 @@ export function glob(pathName: string): Promise<string[]> {
     });
   });
 }
+
+/**
+ * Write contents to file.
+ *
+ * @param {string} filepath
+ * @param {any} data
+ * @returns {Promise<string>}
+ */
+export function write(filepath: string, data: any): Promise<string> {
```

Comment:
```suggestion
export function write(filepath: string, data: any): Promise<void> {
```

Reply:
We need test for this function too.

Reply:
Shall I write the test cases in this PR itself?

Reply:
Yes.